ui/components/app/hold-to-reveal-button/hold-to-reveal-button.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import HoldToRevealButton from './hold-to-reveal-button';
+
+describe('HoldToRevealButton', () => {
+  const onLongPressed = jest.fn();
+
+  it('should render a button with label', () => {
+    const { getByText } = render(
+      <HoldToRevealButton
+        buttonText="Hold to reveal SRP"
+        onLongPressed={onLongPressed}
+      />,
+    );
+    expect(getByText('Hold to reveal SRP')).toBeInTheDocument();
+  });
+
+  it('should render a button when mouse is down and up', () => {
```

Comment:
Any other tests that need to be written here? What happens if you "long press" for the duration of the animation? What happens if you "long press" and lift off before the animation concludes? etc.

Reply:
Fixed in: 270557aff1b25a0ce0135093286afd8739462c8d